### DISCUSSION TITLE
Preview not working

### DISCUSSION THREAD

**Discussion by ArtMin96** [awcodes/filament-curator#333] [Q&A] [answered]
In local everything is normal, but in production the preview does not work.
/storage/media/****** works
/curator/media/****** not working
![image](https://github.com/awcodes/filament-curator/assets/29732308/7a2df2d8-022e-463c-9ec1-9ddf2f0126e1)
![image](https://github.com/awcodes/filament-curator/assets/29732308/95e11d7d-35b1-4df2-9241-010b1494eac2)

**Comment by howdu** [comment, marked as answer]
If you're using nginx try adding a location block to ensure PHP is being served. Nginx tries to efficiently serve know extensions without PHP.

```
location ^~ /curator {
   try_files $uri $uri/ /index.php?$query_string;
}
```
Add it above your `location ~ \.php$`

**Reply by ArtMin96** [reply]
Thanks @howdu

**Reply by arisawali2014** [reply]
i try this and still got an error on the curator

**Reply by arisawali2014** [reply]
Fixed:
```
location ^~ /curator {
   try_files $uri $uri/ /index.php?$query_string;
}
```

**Reply by zulmarij** [reply]
this work for me

```
location ~* ^/(?!curator).*\.(css|js|jpg|jpeg|gif|png|ico|gz|svg|svgz|ttf|otf|woff|woff2|eot|mp4|ogg|ogv|webm|webp|zip|swf|map)$ {
    add_header Access-Control-Allow-Origin "*";
    expires max;
    access_log off;
}
```
